conf/layer.conf

```diff
@@ -30,5 +30,5 @@ LICENSE_PATH += "${LAYERDIR}/licenses"
 # Enable early start of Thunder
 DISTRO_FEATURES_append = " compositor thunder"
```

Comment:
Could you please move this to conf/distro/include/thunder_generic.conf? This file should be included from here by default

Reply:
done